### DISCUSSION TITLE
Decoded file having twice the expected size : Using H266 encoder/decoder from Fraunhofer

### DISCUSSION THREAD

**Discussion by remul19** [fraunhoferhhi/vvdec#141] [Q&A] [answered]
Hi All,

I am completely new to GitHub. So please excuse, if there is any mistake.

Using Fraunhofer h266 encoder [https://github.com/fraunhoferhhi/vvenc](url) , I converted a YUV input (qcif, 176x144, yuv_420) to h266 compressed file. And then using the Fraunhofer decoder [https://github.com/fraunhoferhhi/vvdec](url) , I converted this compressed file back to YUV format. The decoded output YUV file is having **twice the size** of YUV file which was used as the encoder input. Ideally I was expecting that both the YUV files will be having the same size.

The commands used for encoding and decoding is given below.

```
 vvencapp --preset medium -i input.yuv -s 176x144 -r 24 -o input_compressed.266
 vvdecapp -b input_compressed.266 -o output_decoded.yuv
```
 
Size of **output_decoded.yuv** is twice that of **input.yuv**. When played on YUV player, the decoded YUV file is not playing as expected.  During encoding I tried changing the frame rate, but it did not help. Can someone please guide me to resolve this issue ?

**Comment by adamjw24** [comment, marked as answer]
The output of the vvencapp is per default a bitstream with 10bit bit-depth, which results in your decoded yuv being in the yuv420p10le, not yuv420p as your input. Use `--internal-bitdepth 8` during encoding to keep the output format as `yuv420p`

**Reply by adamjw24** [reply]
See also: https://github.com/fraunhoferhhi/vvenc/issues/26

**Reply by remul19** [reply]
Thank you @adamjw24   ... 
Your solution is working good for me.
